### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by au1994**
@vaidik
@vishesh92

### CODE REVIEW

File: movie.sh [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+#Scan input directory from terminal
+
+echo 'enter the directory in which movies are residing'
+read dir
+
+#Get absolute path from the input directory
+path=$(find ~/ -name $dir)
+
+#Make a new file in tmp folder
+touch /tmp/sorted.txt || { echo "Failed to create temp file"; exit 1; }
```
Comment (vaidik) [inline movie.sh:11]: Good one.

Comment (vaidik) [inline movie.sh:11]: to make it more readable, you can write it like this:

```
touch /tmp/sorted.txt || {
    echo "Failed to create temp file";
    exit 1;
}
```

File: movie.sh [hunk 2]
```diff
@@ -1,6 +1,50 @@
...
+
+#Empty the sorted folder
+> /tmp/sorted.txt
+
+#Declaration of variables
+OUT="/tmp/sorted.txt"
+SPACE="   ";
+NA="NA";
+NULL="null";
```
Comment (vaidik) [inline movie.sh:20]: For constants, prefer using `declare`. You can find examples in the style guide.

File: movie.sh [hunk 3]
```diff
@@ -1,6 +1,50 @@
...
+path=$(find ~/ -name $dir)
+
+#Make a new file in tmp folder
+touch /tmp/sorted.txt || { echo "Failed to create temp file"; exit 1; }
+
+#Empty the sorted folder
+> /tmp/sorted.txt
+
+#Declaration of variables
+OUT="/tmp/sorted.txt"
```
Comment (vaidik) [inline movie.sh:17]: What do you need this file for?

File: movie.sh [hunk 4]
```diff
@@ -1,6 +1,60 @@
 #!/bin/bash
-echo 'please input the movie name'
-read name
-echo $name
-var=$(curl -s http://www.omdbapi.com/\?i\=\&t\=$name | jq -r '.imdbRating')
-echo $var
+#Scan input directory from terminal
+
+
+if [ -z $1 ];
```
Comment (vishesh92) [inline movie.sh]: https://google.github.io/styleguide/shell.xml?showone=Pipelines#Loops

File: movie.sh [hunk 5]
```diff
@@ -1,6 +1,60 @@
...
+NULL="null";
+EMPTY="";
+ZERO="0"; 
+
+#Print the heading first
+printf "%-10s  %-20s  %-20s\n" "RATING" "MOVIE" "TITLE"
+
+for f in $path/*
+  do
+     #strip initial path value to get the movie name
```
Comment (vishesh92) [inline movie.sh:27]: The number of spaces here is 5. Be consistent throughout the project. 2 spaces are preferred.

File: movie.sh [hunk 6]
```diff
@@ -1,6 +1,50 @@
...
+SPACE="   ";
+NA="NA";
+NULL="null";
+EMPTY="";
+ZERO="0"; 
+
+
+for f in $path/*
+  do
+     #strip initial path value to get the movie name
```
Comment (vishesh92) [inline movie.sh:27]: You have used 5 spaces here. Be consistent throughout the project. 2 spaces are preferred.

**Pull request merged**